caps/types.go

```diff
@@ -60,6 +63,24 @@ func (t *BoolFileType) Sanitize(c *Capability) error {
 	return nil
 }
 
+// SecuritySnippet for bool-file capability type.
+// Consumers gain permission to read, write and lock the designated file.
+func (t *BoolFileType) SecuritySnippet(c *Capability, securitySystem string) (string, error) {
+	switch securitySystem {
+	case SecurityApparmor:
+		// TODO: switch to the absolute path later
+		path := c.Attrs["path"]
+		// Allow read,write and lock on the file designated by the path.
+		return fmt.Sprintf("%s rwl,\n", path), nil
+	case SecuritySeccomp:
+		return "", nil
+	case SecurityDBus:
+		return "", nil
+	default:
+		return "", fmt.Errorf("unknown security system %q", securitySystem)
```

Comment:
This should be ErrUnknownSecurity so we can differentiate the capability blowing up with a known security system for whatever reason from it not being able to handle the given security system at all.